bindings/matrix-sdk-ffi/src/authentication_service.rs

```diff
@@ -10,6 +10,7 @@ use super::{client::Client, client_builder::ClientBuilder, RUNTIME};
 
 pub struct AuthenticationService {
     base_path: String,
+    passphrase: Option<String>,
```

Comment:
This is a bit of a futile cause considering that we pass this over FFI, but this should be zeroized once the struct is dropped.